test-suite/js/src/test/scala/org/scalajs/testsuite/niobuffer/ByteBufferJSTest.scala

```diff
@@ -0,0 +1,36 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.niobuffer
+
+import org.junit.{Assume, BeforeClass}
+import org.scalajs.testsuite.niobuffer.BufferFactory.ByteBufferFactory
+import org.scalajs.testsuite.utils.Platform
+
+object AllocDirectByteBufferJSTest extends SupportsTypedArrays
+
+class AllocDirectByteBufferJSTest extends ByteBufferTest {
+
+  val factory: ByteBufferFactory =
```

Comment:
I suggest removing the blank line in all those tiny classes that declared only a `val`. It would make it easier to visually group the class wrt to other top-level entities in the file.